routes/index.js

```diff
@@ -1,16 +1,26 @@
 var express = require('express');
 var router = express.Router();
+var session = require('express-session');
```

Comment:
You don't need to require the express-session here in the routes file because it is already in the app.js file.